docs/en/toolkit/learning-resources/guides/installing_app.md

```diff
@@ -234,6 +234,18 @@ settings.tk-maya.project:
  
 You will notice the **Screening Room, Shotgun Panel, and Workfiles2** app’s location identifiers are listed in a different included file and accessed differently than the **About** app. To keep things tidy, these apps were split off to the included settings folder because they have additional settings.
 
+**NOTE:** The python console app already exists in the default configuration, however if you are adding an app that has never been added to your configuration before or if you have changed the version of an app, and you are using a [centralized configuration](https://developer.shotgunsoftware.com/tk-core/initializing.html#centralized-configurations), then there is an additional step you need to take. Open your terminal and browse to where your cloned configuration is stored. From your cloned configuration’s root folder, run the following command:
+
+On Linux or Mac:
+
+`./tank cache_apps`
+
+On Windows:
+
+`tank.bat cache_apps`
+
+This will scan your configuration for apps, engines and frameworks and ensure that they are all downloaded.
+
```

Comment:
I wonder if this should be down at the bottom as an Advanced Topic rather than within the steps of the guide?

Reply:
It can do, I guess as long as the guide doesn't require it, ie the version the user enters isn't different to the one that's already cached then it should be OK. I think ideally we should have picked something like the starter app as that would have been a more complete example?
My concern is that it will get missed at the bottom, do you link to the advanced section at all?

Reply:
fair enough. let's put it in this format:

{% include info title="Note" content="This guide is based on the `tk-config-default2` pipeline configuration. If your config was modified, the location of files, folders, and blocks of YAML settings may vary from what is described here." %}

Reply:
I've made the changes, please would you be able to check that I've encapsulated all of the correct details in the note boxes. I was especially unsure about the last one near the end.

Reply:
Looks really good! That last note rendered especially nicely! Thanks for doing that. all good to me.